src/Form/Control/ScopeBuilder.php

```diff
@@ -200,6 +218,10 @@ class ScopeBuilder extends Control
             self::OPERATOR_EQUALS => Condition::OPERATOR_EQUALS,
             self::OPERATOR_DOESNOT_EQUAL => Condition::OPERATOR_DOESNOT_EQUAL,
         ],
+        'lookup' => [
+            self::OPERATOR_EQUALS => Condition::OPERATOR_EQUALS,
+            self::OPERATOR_DOESNOT_EQUAL => Condition::OPERATOR_DOESNOT_EQUAL,
```

Comment:
ah I see - it's like that in Condition class too. Then probably it's correct to have it like this here too.